### PR TITLE
APIFunction changes for print log

### PR DESCRIPTION
**Pull request by 1234momo**
Added 2 functions into APIFunction.js for adding a print request and getting all the print requests

### CODE REVIEW

File: src/APIFunctions/2DPrinting.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import axios from 'axios';
-import { PrintApiResponse } from './ApiResponses';
-import { RPC_API_URL } from '../config/config.json';
+import {
+  PrintApiResponse,
+  ApiResponse } from './ApiResponses';
```
Comment (hsorathia) [inline src/APIFunctions/2DPrinting.js:4]: have ApiResponse on it's own line so it looks like
```js
import {
  PrintApiResponse,
  ApiResponse
} from './ApiResponses';
```

Comment (evanugarte) [reply src/APIFunctions/2DPrinting.js:4]:
```suggestion
import {
  PrintApiResponse,
  ApiResponse
} from './ApiResponses';
```

File: src/APIFunctions/2DPrinting.js [hunk 2]
```diff
@@ -62,6 +65,37 @@ export async function printPage(data) {
...
+/**
+ * Log the print request
+ * @param {Object} data         Encoded file
+ * @returns {PrintApiResponse}  Containing information for if
+ *                              the page is printing
+ */
+export async function logPrintRequest(data) {
+  let status = new ApiResponse();
+  await axios.post(LOGGING_API_URL + '/api/PrintLog/addPrintLog', data)
+    .catch(() => {
```
Comment (aashrav) [inline src/APIFunctions/2DPrinting.js:77]: If the routing doesn't throw an error what will this function return? I don't see status.responseData being set anywhere

File: src/APIFunctions/2DPrinting.js [hunk 3]
```diff
@@ -62,6 +65,37 @@ export async function printPage(data) {
...
+      status.error = true;
```
Comment (aashrav) [inline src/APIFunctions/2DPrinting.js:78]: If the routing doesn't throw an error what will this function return? I don't see status.responseData being set anywhere

Comment (jerrylee17) [reply src/APIFunctions/2DPrinting.js:78]: I don't think there is any responseData needed. It just returns an ApiResponse with no error

**Pull request merged**